content/_includes/post.macros.njk

```diff
@@ -236,7 +236,7 @@ params:
 {# @docs
 label: upcoming
 category: events
-note: Listof upcoming events from a post
+note: List of upcoming events from a post
```

Comment:
I'm not clear what works best here... I wondering it if makes sense to add a `past` event macro, or it makes more sense to use a conditional to check if it is future, and if not, show the past event dates with a heading or what not. Any thoughts as far as the approach @mirisuzanne @dvdherron?